src/ClusterManager/job_launcher.py

```diff
@@ -663,19 +658,31 @@ def submit_job_impl(self, job):
             jobMeta["LaunchCMD"] = pods[0].spec.containers[0].command
 
             jobMetaStr = base64.b64encode(json.dumps(jobMeta))
-            dataHandler.UpdateJobTextField(job_object.job_id, "jobMeta", jobMetaStr)
+
+            dataFields = {
+                "jobStatus": "scheduling",
+                "jobDescriptionPath": job_description_path,
+                "jobDescription": base64.b64encode(job_description),
+                "lastUpdated": datetime.datetime.now().isoformat(),
+                "jobMeta": jobMetaStr
+            }
+            conditionFields = {"jobId": job_object.job_id}
+            dataHandler.UpdateJobTextFields(conditionFields, dataFields)
```

Comment:
job manager also have a lot of these calls, I will change to this call after you merged